src/presentation/controller/registration/LimitedPartnershipController.ts

```diff
@@ -117,17 +124,25 @@ class LimitedPartnershipController extends AbstractController {
     return async (request: Request, response: Response, next: NextFunction) => {
       try {
         const { tokens, ids } = super.extract(request);
+        const closeTransactionResponse = await this.limitedPartnershipService.closeTransaction(tokens, ids.transactionId);
+        const startPaymentSessionUrl: string = closeTransactionResponse.headers?.["x-payment-required"];
 
-        // TODO Use the response from this call to get hold of the payment URL, when the
-        //      payment journey is implemented
-        //
-        //      E.g.   apiResponse.headers?.["x-payment-required"];
-        await this.limitedPartnershipService.closeTransaction(tokens, ids.transactionId);
+        if (!startPaymentSessionUrl) {
+          throw new Error("No payment URL found in the response");
```

Comment:
can you change message to say what was called, "No payment URL found in response header from closeTransaction"